geoips/geoips_utils.py

```diff
@@ -29,10 +29,16 @@
 
 def get_entry_point_group(group):
     """Get entry point group."""
+    # NOTE: When there is a .egg-info directory in the plugin package top
+    # level (ie, from setuptools pip install -e), it seems to return that
+    # package twice in this list.  For now, just use the full list with
+    # duplicates.
     if sys.version_info[:3] >= (3, 10, 0):
-        return metadata.entry_points(group=group)
+        eps = metadata.entry_points(group=group)
     else:
-        return metadata.entry_points()[group]
+        eps = metadata.entry_points()[group]
```

Comment:
@jsolbrig This is where yours was erroring on python 3.9?  I think that must be a red herring, because that else is there specifically to support 3.9, there must be something else causing 3.9 to fail.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/419#issuecomment-1924466402

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/419#issuecomment-1934885067
I think we are good to go with removing support for 3.9 at this point. We will deal with additional errors as they arise.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/419#issuecomment-1934875033
Note there may be issues with CLAVR-x build, etc, but we will deal with those separately (that is an external package, not actually part of the geoips plugin ecosystem)

Reply:
Yes, this is where my code was failing on 3.9. Should we create an issue to remove any references to 3.9? There are references elsewhere, too, especially in the CI.

Reply:
Created an issue relating to this. Resolving for now.